### ISSUE TITLE
Questions about test results and data preprocessing

### ISSUE THREAD

**Issue by JY00002** [daothanh2011/HAMCNet#1]
Thank you for sharing the code, but after my testing and analysis, I did not reproduce the results obtained in your readme. In addition, I found a problem in the data processing process, and I analyzed your source code. You normalized both the training and test sets by class and SNR, but the problem is that for the test set, you can't possibly know the class and SNR of the data, which is a paradox.
In addition, regarding the division of the data set, your test results cannot be reproduced, because you do not have fixed random numbers in the process of dividing the data set, and the training data will completely overflow into the test set, resulting in unreliable reproduced results. .
I hope you take the rigor of your code seriously. Finally, thanks a lot for the code you shared.

**Comment by daothanh2011** [comment]
I appreciate you testing the code.
- As you can see in the README, I got these results when training the separate datasets for training and testing. As a result, these results look slightly different when executing the network on eval.py code (I should separate the training and testing datasets for both train.py and eval.py code. I will fix the dataset for the next code push).
- Regarding your concern about the SNR, the data feeding to the network is not aware of the class or SNR. Thus, running the network without them is not a problem. The neural network must be tested at a specific SNR in order to obtain a confusion matrix. As a result, the SNRs were used for the prediction of the network on those SNRs.
Hope it makes sense to you.
